tests/nicerdicer.test.ts

```diff
@@ -0,0 +1,26 @@
+import nicerdicer from "@/nicerdicer";
+
+describe("nicerdicer", () => {
+  it("should return a number from 1-6", () => {
+    const randomNumber = nicerdicer();
+
+    expect(randomNumber).toBeGreaterThanOrEqual(1);
+    expect(randomNumber).toBeLessThanOrEqual(6);
+  });
+
+  it("should increment the counter for the generated number", () => {
+    const counters: number[] = [0, 0, 0, 0, 0, 0];
+
+    for (let i = 0; i < 100; i++) {
+      const randomNumber = nicerdicer();
+      counters[randomNumber - 1]++;
+    }
+
+    for (let i = 0; i < 6; i++) {
+      expect(counters[i]).toBeGreaterThan(0);
+    }
```

Comment:
Der Test ist etwas unspezifisch. Mit dem Test würde z.B. nicht geprüft, ob eine Zahl auch wirklich in dem korrekten Container landet.